### PR TITLE
Update flashing instructions for Edison

### LINKED ISSUE #109: Windows instructions for new device don't currently work
The Windows instructions for the Edison ask the user to download the Windows drivers for Edison, but the link is broken, and I could not find another source for the drivers. Does it make sense to change the directions to use the Flash Lite tool? It might be possible to unify these instructions across Windows, Mac, and Linux if that tool works across the board

### PR DESCRIPTION
**Pull request by zwalchuk**
Change the link to the Windows Edison driver to make it current.
Modify the Mac instructions to take into account the possibility of having to give the flash script execute permission.

Fixes #109

### CODE REVIEW

File: edison.coffee [hunk 1]
```diff
@@ -16,6 +16,10 @@ OSX_LINUX_FLASH = '''
 	Execute the following from the unzipped directory:
 	<br>
 	<code>sudo ./flashall.sh</code>
+	<br>
+	If you have trouble executing, you may need to modify the file permissions first:
+	<br>
+	<code>sudo chmod 755 ./flashall.sh</code>
```
Comment (imrehg) [inline edison.coffee]: I think it should be `sudo chmod +x ./flashall.sh`

File: edison.coffee [hunk 2]
```diff
@@ -19,7 +19,7 @@ OSX_LINUX_FLASH = '''
 	<br>
 	If you have trouble executing, you may need to modify the file permissions first:
 	<br>
-	<code>sudo chmod 755 ./flashall.sh</code>
+	<code>sudo chmod +x ./flashall.sh</code>
```
Comment (floion) [inline edison.coffee]: Can you squash this commit in the first one please?

File: edison.coffee (hunk 1 above)
Comment (zwalchuk) [reply edison.coffee]: Fixed

**Comment by floion** [comment]
@resin-jenkins ok to test

**Comment by floion** [comment]
@zwalchuk this needs to get rebased

**Comment by zwalchuk** [comment]
@floion should I be merging this myself, or is that something you do?

**Comment by floion** [comment]
@zwalchuk this PR needs to get rebased. It is not mergeable now because there are conflicts as you can see below

**Comment by zwalchuk** [comment]
@floion I don't see any conflicts, I have the option to merge

**Comment by floion** [comment]
Strange. Then merge it please

**Pull request merged**